department-of-veterans-affairs/va.gov-cms
# VACMS-10281 Changes Facilities view table caption

## Description

Closes #10281 

## Testing done

Manually

## Screenshots
![facilities1-Screen Shot 2022-09-16 at 3 22 04 PM](https://user-images.githubusercontent.com/766573/190810386-daee4714-8c09-466e-ae51-0cddd84c73b9.png)
![facilities2-Screen Shot 2022-09-16 at 3 17 07 PM](https://user-images.githubusercontent.com/766573/190810422-2dcfa0b2-1bde-4c01-b7d2-eb78c9d8823b.png)



## QA steps
As an admin user:
Visit Facilities Views
- /admin/content/facilities
  - Validate that the caption is "List of facilities"
- /admin/content/facilities/addresses
  - Validate that the caption is "List of facility addresses"
- /admin/content/facilities/facility-status
  - Validate that the caption is "List of facilities and their status"
- /admin/content/facilities/vamc-facility-health-services
  - Validate that the caption is "List of health services at VAMC facilities"
- /admin/content/facilities/vamc-facility-non-clinical-services
  - Validate that the caption is "List of non-clinical services at VAMC facilities"
- /admin/content/facilities/vamc-prepare-for-visit
  - Validate that the caption is 'List of accordion items in the "Prepare for your visit" field on VAMC facilities'
- /admin/content/facilities/vamc-systems
  - Validate that the caption is "List of VAMC systems"
- /admin/content/facilities/vamc-system-service-audit
  - Validate that the caption is "List of health services at VAMC systems"
- /admin/content/facilities/vet-center-services
  - Validate that the caption is "List of services at Vet Centers"
 
Visit Non-facilities Content Views
Example views:
- /admin/content/audit
- /admin/content/bulk
- /admin/content/va-forms

None of these tables should have a caption

### Definition of Done


### Select Team for PR review




### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
